examples/gogo/command/src/main/java/org/osgi/enroute/examples/gogo/command/ComponentMulti.java

```diff
@@ -0,0 +1,28 @@
+
+package org.osgi.enroute.examples.gogo.command;
+
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+        /* Activate the component immediately, also if no one depends on it. */
+        immediate = true,
+        /* You need to implement some class, without any it will not be picked up. */
+        service = { ComponentMulti.class },
```

Comment:
This sort of comment should be included in the documentation page on the https://enroute.osgi.org site, not in the source file.

Reply:
I understand.